source/frontend/src/features/mapSideBar/lease/tabs/consultations/edit/ConsultationUpdateContainer.tsx

```diff
@@ -0,0 +1,124 @@
+import axios, { AxiosError } from 'axios';
+import { FormikHelpers } from 'formik';
+import { useCallback, useEffect, useState } from 'react';
+import { useHistory, useLocation } from 'react-router-dom';
+import { toast } from 'react-toastify';
+
+import { useOrganizationRepository } from '@/features/contacts/repositories/useOrganizationRepository';
+import { usePersonRepository } from '@/features/contacts/repositories/usePersonRepository';
+import { useConsultationProvider } from '@/hooks/repositories/useConsultationProvider';
+import { IApiError } from '@/interfaces/IApiError';
+import { ApiGen_Concepts_ConsultationLease } from '@/models/api/generated/ApiGen_Concepts_ConsultationLease';
+import { exists, isValidId } from '@/utils';
+
+import { LeasePageNames } from '../../../LeaseContainer';
+import { IConsultationEditFormProps } from './ConsultationEditForm';
+import { ConsultationFormModel } from './models';
+
+export interface IConsultationUpdateContainerProps {
+  leaseId: number;
+  consultationId: number;
+  onSuccess: () => void;
+  View: React.FunctionComponent<React.PropsWithChildren<IConsultationEditFormProps>>;
+}
+
+const ConsultationUpdateContainer: React.FunctionComponent<
+  React.PropsWithChildren<IConsultationUpdateContainerProps>
+> = ({ leaseId, consultationId, onSuccess, View }) => {
+  const [leaseConsultations, setLeaseConsultations] = useState<ApiGen_Concepts_ConsultationLease[]>(
```

Comment:
some more unused variables detected by GH.

Reply:
I think this is an earlier commit